Author: Alessio Treglia <alessio@debian.org>
Description: Replace useradd with adduser to comply with Debian Policy.
Bug-Ubuntu: https://launchpad.net/bugs/810907
Forwarded: no
---
 src/daemon.c |   13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

--- accountsservice.orig/src/daemon.c
+++ accountsservice/src/daemon.c
@@ -1110,12 +1110,13 @@ daemon_create_user_authorized_cb (Daemon
 
         sys_log (context, "create user '%s'", cd->user_name);
 
-        argv[0] = "/usr/sbin/useradd";
-        argv[1] = "-m";
-        argv[2] = "-c";
-        argv[3] = cd->real_name;
-        argv[4] = cd->user_name;
-        argv[5] = NULL;
+        argv[0] = "/usr/sbin/adduser";
+        argv[1] = "--quiet";
+        argv[2] = "--disabled-login";
+        argv[3] = "--gecos";
+        argv[4] = cd->real_name;
+        argv[5] = cd->user_name;
+        argv[6] = NULL;
 
         error = NULL;
         if (!spawn_with_login_uid (context, argv, &error)) {
